apps/ledger-live-mobile/e2e/specs/nftGallery.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { expect } from "detox";
+import PortfolioPage from "../models/wallet/portfolioPage";
+import { loadConfig } from "../bridge/server";
+import { tapByElement } from "../helpers";
+import WalletTabNavigatorPage from "../models/wallet/walletTabNavigator";
+import NftViewerPage from "../models/nft/nftViewerPage";
+import NftGalleryPage from "../models/wallet/nftGalleryPage";
+
+let portfolioPage: PortfolioPage;
+let walletTabNavigatorPage: WalletTabNavigatorPage;
+let nftGalleryPage: NftGalleryPage;
+let nftViewerPage: NftViewerPage;
+
+describe("NFT Gallery screen", () => {
+  beforeAll(async () => {
+    await loadConfig("2_accounts_eth_with_tokens_and_nft", true);
+    portfolioPage = new PortfolioPage();
+    walletTabNavigatorPage = new WalletTabNavigatorPage();
+    nftGalleryPage = new NftGalleryPage();
+    nftViewerPage = new NftViewerPage();
+  });
+
+  it("should open on Portofolio page", async () => {
+    await expect(portfolioPage.getSettingsButton()).toBeVisible();
+  });
+
+  it("should see NFT tab", async () => {
+    await expect(walletTabNavigatorPage.getNftGalleryTab()).toBeVisible();
+  });
+
+  it("should navigate to NFT gallery on NFT tab press", async () => {
+    await tapByElement(walletTabNavigatorPage.getNftGalleryTab());
```

Comment:
Instead of importing the `tapByElement()` helper method in the test file, you should have a method in the `walletTabNavigatorPage` like `navigateToNftGalleryTab()`, which uses the `tapByElement()` function. This is more of the Page Object Model convention and helps reduce the imports we do in the test file

Reply:
what's the problem with doing imports in the test file ?
But ok yes i can this put it in Page class

Reply:
As a convention we try to keep the common actions like click, tap, type text, etc in the Page Object Model files. So test files should be more English sentences like `nftPage.navigateToNft()` rather than `tapElement(nftPage.webElement())`. It's not a massive difference but it helps to keep things neat and consistent

Reply:
It also helps keep the tests focused on user behaviour. A user doesn't care about tapping a series of elements, they just want to see their NFTs in this case